[2025-07-31T12:25:09.354+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-31T12:25:09.365+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: world_population_ETL_dag_43.retrieve_rows_task manual__2025-07-31T12:24:56.572054+00:00 [queued]>
[2025-07-31T12:25:09.370+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: world_population_ETL_dag_43.retrieve_rows_task manual__2025-07-31T12:24:56.572054+00:00 [queued]>
[2025-07-31T12:25:09.370+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-07-31T12:25:09.377+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): retrieve_rows_task> on 2025-07-31 12:24:56.572054+00:00
[2025-07-31T12:25:09.386+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=667) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-31T12:25:09.387+0000] {standard_task_runner.py:72} INFO - Started process 669 to run task
[2025-07-31T12:25:09.386+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'world_population_ETL_dag_43', 'retrieve_rows_task', 'manual__2025-07-31T12:24:56.572054+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/project_01_dag.py', '--cfg-path', '/tmp/tmp1st6wy_4']
[2025-07-31T12:25:09.388+0000] {standard_task_runner.py:105} INFO - Job 41: Subtask retrieve_rows_task
[2025-07-31T12:25:09.417+0000] {task_command.py:467} INFO - Running <TaskInstance: world_population_ETL_dag_43.retrieve_rows_task manual__2025-07-31T12:24:56.572054+00:00 [running]> on host 1e069286f646
[2025-07-31T12:25:09.471+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='givi-abe' AIRFLOW_CTX_DAG_ID='world_population_ETL_dag_43' AIRFLOW_CTX_TASK_ID='retrieve_rows_task' AIRFLOW_CTX_EXECUTION_DATE='2025-07-31T12:24:56.572054+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-31T12:24:56.572054+00:00'
[2025-07-31T12:25:09.472+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-31T12:25:09.486+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-31T12:25:09.493+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-31T12:25:09.493+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=world_population_ETL_dag_43, task_id=retrieve_rows_task, run_id=manual__2025-07-31T12:24:56.572054+00:00, execution_date=20250731T122456, start_date=20250731T122509, end_date=20250731T122509
[2025-07-31T12:25:09.520+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-31T12:25:09.532+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-31T12:25:09.533+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
